packages/@sanity/desk-tool/src/panes/documentPane/timeline/timelineMenu.tsx

```diff
@@ -0,0 +1,137 @@
+import {useTimeAgo} from '@sanity/base/hooks'
+import {Chunk} from '@sanity/field/diff'
+import {SelectIcon} from '@sanity/icons'
+import {useClickOutside, Button, Box, Popover, Card} from '@sanity/ui'
+import {upperFirst} from 'lodash'
+import React, {useCallback, useState, useMemo} from 'react'
+import styled from 'styled-components'
+import {useDocumentHistory} from '../documentHistory'
+import {sinceTimelineProps, revTimelineProps, formatTimelineEventLabel} from './helpers'
+import {Timeline} from './timeline'
+
+interface TimelineMenuProps {
+  chunk: Chunk | null
+  mode: 'rev' | 'since'
+}
+
+const Root = styled(Popover)`
+  & > div {
+    display: flex;
+    flex-direction: column;
+
+    & > [data-ui='Card'] {
+      flex: 1;
+      min-height: 0;
+      display: flex;
+      flex-direction: column;
+
+      /* This is the scrollable container rendered by <Timeline /> */
+      & > div {
+        flex: 1;
+        min-height: 0;
+      }
+    }
+  }
+`
+
+export function TimelineMenu({chunk, mode}: TimelineMenuProps) {
+  const [open, setOpen] = useState(false)
+  const [buttonRef, setButtonRef] = useState(null)
+  const [menuContent, setMenuContent] = useState(null)
+
+  const {historyController, setRange, setTimelineMode, timeline} = useDocumentHistory()
+
+  const handleOpen = () => {
+    setTimelineMode(mode)
+    setOpen(true)
+  }
+
+  const handleClose = useCallback(() => {
+    setTimelineMode('closed')
+    setOpen(false)
+  }, [setTimelineMode])
+
+  const handleClickOutside = useCallback(() => {
+    handleClose()
+  }, [handleClose])
+
+  useClickOutside(handleClickOutside, [menuContent, buttonRef])
+
+  const selectRev = useCallback(
+    (revChunk: Chunk) => {
+      const [sinceId, revId] = historyController.findRangeForNewRev(revChunk)
+      setTimelineMode('closed')
+      setOpen(false)
+      setRange(sinceId, revId)
+    },
+    [historyController, setRange, setTimelineMode]
+  )
+
+  const selectSince = useCallback(
+    (sinceChunk: Chunk) => {
+      const [sinceId, revId] = historyController.findRangeForNewSince(sinceChunk)
+      setTimelineMode('closed')
+      setOpen(false)
+      setRange(sinceId, revId)
+    },
+    [historyController, setRange, setTimelineMode]
+  )
+
+  const loadMoreHistory = useCallback(
+    (state: boolean) => {
+      historyController.setLoadMore(state)
+    },
+    [historyController]
+  )
+
+  const content = open && (
+    <Card ref={setMenuContent as any} radius={3}>
```

Comment:
This should simply be a `<div>` instead of `<Card>` so the background won’t conflict with the rounded corners of  the `Popover`.